quiz.css

```diff
@@ -0,0 +1,57 @@
+
+.product {
+  border: 1px solid lightblue;
+  border-radius: 5px;
+  margin:.75em;
+  padding: 10px;
+  font-family: 'Lora', serif;
+  clear: 
+}
+
+.product h1 {
+  border-bottom: 1px dotted lightgrey;
+  text-align: center;
+  font-family: 'Open Sans', sans-serif;
+  padding-bottom: 15px;
+}
+
+.resize{
+    width:100%; 
+    display: block;
+    height: auto;
+    margin: auto;
+}
+
+
+.description {
+  text-align: justify;
+}
+
+.pricing p {
+  font-family: 'Lora', serif;
+  font-weight: 700;
+}
+
+.availability {
+  background: grey;
+  color:yellow;
+}
+
+ .product footer: {
+  font-size: 50%;
+}
+
+@media (min-width: 600px) {
+  body {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Hey Scott, I'm having problems using a float. I changed it all to floats, but then it created a really strange margin on the left side of the third card. All other changes are fine, and have been implemented.